examples/05_stable_diffusion/scripts/download_pipeline.py

```diff
@@ -15,16 +15,19 @@
 import click
 import torch
 from diffusers import StableDiffusionPipeline
+from aitemplate.utils.import_path import import_parent
+if __name__ == "__main__":
+    import_parent(filepath=__file__, level=1)
+from src.compile_lib.util import get_work_dir_location_diffusers
 
 
 @click.command()
 @click.option("--token", default="", help="access token")
-@click.option(
-    "--save_directory",
-    default="./tmp/diffusers-pipeline/stabilityai/stable-diffusion-v2",
-    help="pipeline files local directory",
-)
-def download_pipeline_files(token, save_directory) -> None:
+
+def download_pipeline_files(token) -> None:
```

Comment:
Seems like the linter doesn't like the empty line

Reply:
Done